microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -791,6 +791,10 @@ def _install_fedora_dependencies(self) -> None:
         # update to latest kernel before instaling dependencies
         rhel.install_packages("kernel")
         node.reboot()
+        if isinstance(node.os, Redhat):
+            # rhel is not guaranteed to have a DPDK package in the default
+            # repos, add EPEL to get a DPDK version in the pkg manager.
+            node.os.install_epel()
```

Comment:
is it 'node.os.enable_extras()'?

Reply:
ah sorry I should have moved this to a draft, I was just editing and testing. I'll promote out of draft status when it's ready for review again.